anndata/_core/anndata.py

```diff
@@ -138,18 +138,15 @@ class AnnData(metaclass=utils.DeprecationMixinMeta):
     variables :attr:`var` (:attr:`varm`, :attr:`varp`),
     and unstructured annotations :attr:`uns`.
 
-    .. figure:: https://falexwolf.de/img/scanpy/anndata.svg
-       :width: 350px
+    .. figure:: ../_static/img/anndata_schema.svg
+       :width: 260px
```

Comment:
the font looks a bit wonky for me:

![grafik](https://user-images.githubusercontent.com/291575/146348421-2f3e71a5-f8b3-4d62-b328-daa75c0a6abd.png)

Reply:
The PDF looks fine though, could probably go from pdf -> svg